testing/test_doctest.py

```diff
@@ -129,6 +129,52 @@ def test_multiple_patterns(self, testdir):
             '*1 passed*',
         ])
 
+    def test_encoding_ascii(self, testdir):
+        """Test support for --doctest-encoding option.
+        """
+        testdir._makefile(".txt", ["""
+            >>> 1
+            1
+        """], {}, encoding='ascii')
+
+        result = testdir.runpytest("--doctest-encoding=ascii")
+
+        result.stdout.fnmatch_lines([
+            '*1 passed*',
+        ])
+
+    def test_encoding_latin1(self, testdir):
+        """Test support for --doctest-encoding option.
+        """
+        testdir._makefile(".txt", ["""
```

Comment:
That would break the code for Python 3.0 I guess because iirc it didn't have the unicode prefix anymore.